website/static/js/pages/project-base-page.js

```diff
@@ -32,4 +32,41 @@ $(document).ready(function() {
     $.getJSON(node.urls.api, function(data) {    
         $('body').trigger('nodeLoad', data);
     });
+
+    var self = this;
+    var thresholdScrollPosition = 50;
+    var smallScreenSize = 767;
+    var nonNavTopMargin = 50;
+    var navMaxTopMargin = 95;
+    self.checkPanelPosition = function() {
+        var bodyWidth = $(document.body).width();
+        var scrollTopPosition = $(window).scrollTop();
+        if (bodyWidth <= smallScreenSize) {
+            if (scrollTopPosition >= thresholdScrollPosition) {
+                $('.cp-handle').css('margin-top', nonNavTopMargin);
+            }
+            else {
+                $('.cp-handle').css('margin-top', navMaxTopMargin - scrollTopPosition);
+            }
+        }
+    };
+    var THROTTLE = 10;
+    self.debouncePanelPosition = $osf.debounce(function() {
```

Comment:
For accuracy, this can be renamed to `debouncedPanelPosition`, though i'm not even sure it's necessary; you should be able to do

```js
$(window).scroll($osf.debounce(self.checkPanelPosition, THROTTLE));
```